lang/py/avro/test/test_schema.py

```diff
@@ -505,6 +505,205 @@ class InvalidTestSchema(TestSchema):
     ),
 ]
 
+"""
+Fingerprint examples are in the form of tuples:
```

Comment:
Since this isn't anchored to a module or function, it doesn't do much good as a string -- probably better to make it a comment.